### PR TITLE
innerCode should be run through Prismjs as well (try 2)

### PR DESCRIPTION
**Pull request by docwhat**
This is the second attempt at issue #4556

In addition, it fixes issue #4597

### CODE REVIEW

File: packages/gatsby-remark-prismjs/src/highlight-code.js [hunk 1]
```diff
@@ -10,7 +10,11 @@ module.exports = (language, code, lineNumbersHighlight = []) => {
       loadPrismLanguage(language)
     } catch (e) {
       // Language wasn't loaded so let's bail.
-      return code
+      if (_.includes([`plain`, `plaintext`, `text`], language)) {
+        return _.escape(code)
```
Comment (KyleAMathews) [inline packages/gatsby-remark-prismjs/src/highlight-code.js]: Thanks! I'd noticed before that html in code samples sometimes would render as HTML which was confusing — why we didn't catch it before I think was that remark normally escapes code inside inline code and blocks so this problem doesn't happen. Also most code blocks do have languages set.

**Comment by docwhat** [comment]
> It looks like there's some TravisCI problems.

They should be fixed.  I'm still trying to understand the test tools.

I just discovered: `yarn run test:watch --testPathPattern gatsby-remark-prism` which is helpful.

**Comment by KyleAMathews** [comment]
Thanks!

**Comment by KyleAMathews** [comment]
It just occurred to me — we should probably *always* escape code. That's probably why there's some buttons which show up at the top of page briefly on the first part of the tutorial https://www.gatsbyjs.org/tutorial/part-one/

@docwhat any reason you can not to always escape code?

**Comment by docwhat** [comment]
I can reverse the check so it doesn’t escape is and only if it is set to `language-none`. How about that?

**Comment by docwhat** [comment]
@KyleAMathews done in PR #4620

**Comment by KyleAMathews** [comment]
I guess the question is does prismjs escape html entities by default. Which I think it does so what's happening on the tutorial is probably something different.

**Comment by docwhat** [comment]
Correct.

Maybe, if you give me the url and a description of the problem, I can take a look.

**Comment by KyleAMathews** [comment]
https://github.com/gatsbyjs/gatsby/pull/4599#issuecomment-374411895 — load it and look for some buttons to appear at the top than disappear.

**Comment by docwhat** [comment]
I don't see any disappearing buttons at the top. :-(

**Pull request merged**